### PR TITLE
HOTT-4261: Adds seasonal measures worksheet

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-4261

### What?

I have added/removed/altered:

- [x] Added seasonal measures worksheet
- [x] Fixed some presentational issues with quota coverage and overview worksheets

### Why?

I am doing this because:

- This is require for feature completeness

### CODE REVIEW

File: app/lib/reporting/differences/seasonal.rb [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        acc = []
+
+        each_seasonal_measure do |measure|
+          row = build_row_for(measure)
+
+          acc << row unless row.nil?
+        end
+
+        acc
+      end
```
Comment (alexdesi) [inline app/lib/reporting/differences/seasonal.rb:94]: a more compact for would be:
```
each_seasonal_measure.map{|m| build_row_for(m)}
                                            .compact
```

File: app/lib/reporting/differences/seasonal.rb [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+          yield seasonal_measure if seasonal_measure.failed_duty_status.present?
+        end
+      end
+
+      def applicable_measures
+        @applicable_measures ||= begin
+          measures = Measure
+          .with_seasonal_measures(measure_type_ids, geographical_area_ids)
+          .all
```
Comment (alexdesi) [inline app/lib/reporting/differences/seasonal.rb:125]: indentation does not seem right here

**Review by alexdesi — DISMISSED** [review]
It looks good, minor comments.

File: app/lib/reporting/differences/seasonal.rb (hunk 1 above)
Comment (willfish) [reply app/lib/reporting/differences/seasonal.rb:94]: I think the terser version is harder to read, personally

Comment (willfish) [reply app/lib/reporting/differences/seasonal.rb:94]: I agree actually, I'll change this

**Pull request merged**